bookwyrm/views/landing/register.py

```diff
@@ -55,7 +61,7 @@ def post(self, request):
 
         localname = form.data["localname"].strip()
         email = form.data["email"]
-        password = form.data["password"]
+        password = None if 'oauth-newuser' in request.session else form.data["password"]
```

Comment:
I really don't think that password authentication should be disabled if OAuth is enabled, because that prevents users such as myself using my password manager to automatically authenticate. If it were the case, I would never enable OAuth, whereas I without doubt would otherwise.